index.js

```diff
@@ -0,0 +1,59 @@
+const express = require("express");
+const helmet = require("helmet");
+const knex = require("knex");
+
+const knexConfig = require("./knexfile.js");
+
+const db = knex(knexConfig.development);
+
+const server = express();
+
+server.use(helmet());
+server.use(express.json());
+
+server.get("/api/projects/:id", (req, res) => {
+	const project_id = req.params.id;
+	db("Projects")
+		.where({ project_id: project_id })
```

Comment:
You can use shorthand since the key and value are the same here; `.where({ project_id })`